addons/google_calendar/models/google_calendar.py

```diff
@@ -350,118 +348,112 @@ def get_one_event_synchro(self, cr, uid, google_id, context=None):
         headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
-        data = self.generate_data(cr, uid, oe_event, context=context)
+        data = self.generate_data(oe_event)
+        
         data['sequence'] = google_event.get('sequence', 0)
         data_json = simplejson.dumps(data)
 
-        status, content, ask_time = self.pool['google.service']._do_request(cr, uid, url, data_json, headers, type='PATCH', context=context)
+        status, content, ask_time = self.env['google.service']._do_request(url, data_json, headers, type='PATCH')
 
         update_date = datetime.strptime(content['updated'], "%Y-%m-%dT%H:%M:%S.%fz")
-        calendar_event.write(cr, uid, [oe_event.id], {'oe_update_date': update_date})
+        CalendarEvent.browse([oe_event.id]).write({'oe_update_date': update_date})
 
-        if context['curr_attendee']:
-            self.pool['calendar.attendee'].write(cr, uid, [context['curr_attendee']], {'oe_synchro_date': update_date}, context)
+        if self.env.context['curr_attendee']:
+            self.env['calendar.attendee'].browse([self.env.context['curr_attendee']]).write({'oe_synchro_date': update_date})
 
-    def update_an_event(self, cr, uid, event, context=None):
-        data = self.generate_data(cr, uid, event, context=context)
+    def update_an_event(self, event):
+        data = self.generate_data(event)
 
         url = "/calendar/v3/calendars/%s/events/%s" % ('primary', event.google_internal_event_id)
         headers = {}
-        data['access_token'] = self.get_token(cr, uid, context)
+        data['access_token'] = self.get_token()
 
-        status, response, ask_time = self.pool['google.service']._do_request(cr, uid, url, data, headers, type='GET', context=context)
+        status, response, ask_time = self.env['google.service']._do_request(url, data, headers, type='GET')
         #TO_CHECK : , if http fail, no event, do DELETE ?
         return response
 
-    def update_recurrent_event_exclu(self, cr, uid, instance_id, event_ori_google_id, event_new, context=None):
-        gs_pool = self.pool['google.service']
+    def update_recurrent_event_exclu(self, instance_id, event_ori_google_id, event_new):
+        GsPool = self.env['google.service']
 
-        data = self.generate_data(cr, uid, event_new, context=context)
+        data = self.generate_data(event_new)
 
         data['recurringEventId'] = event_ori_google_id
         data['originalStartTime'] = event_new.recurrent_id_date
 
-        url = "/calendar/v3/calendars/%s/events/%s?access_token=%s" % ('primary', instance_id, self.get_token(cr, uid, context))
+        url = "/calendar/v3/calendars/%s/events/%s?access_token=%s" % ('primary', instance_id, self.get_token())
         headers = {'Content-type': 'application/json'}
 
-        data['sequence'] = self.get_sequence(cr, uid, instance_id, context)
+        data['sequence'] = self.get_sequence(instance_id)
```

Comment:
may be better to update the data dictionary in one go, 
data.update(recurringEventId=event_ori_google_id,
                    originalStartTime=event_new.recurrent_id_date,
                    sequence=self.get_sequence(instance_id))